src/main/java/org/kohsuke/github/GHMarketplaceAccount.java

```diff
@@ -0,0 +1,117 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URL;
+
+/**
+ * A Github Marketplace Account.
+ *
+ * @author Paulo Miguel Almeida
+ * @see GHMarketplaceListAccountBuilder#retrieve()
```

Comment:
```suggestion
 * @see GHMarketplaceListAccountBuilder#createRequest()
```

Changed this in the `GitHub` class, should try to stay consistent.